src/main.rs

```diff
@@ -11,19 +14,43 @@ mod cli;
 #[cfg(feature = "egui")]
 mod egui;
 
+#[derive(Debug, Parser)]
+#[clap(author, version, about)]
+struct Args {
+    #[cfg(feature = "cli")]
+    #[clap(short, long, required = true, help = "The serial(s) you want to search for, in quotes and separated by spaces")]
+    titles: Vec<String>,
+    #[cfg(feature = "cli")]
+    #[clap(short, long, help = "Downloads all available updates printing only errors, without needing user intervention.")]
+    silent: bool,
+    #[cfg(feature = "cli")]
+    #[clap(short, long, help = "Target folder to save the downloaded update files to.")]
+    destination_path: Option<PathBuf>,
+    #[clap(short, long, help = "Disables logging of program output into .log files.")]
```

Comment:
Since it means disabling logging, I'd rather have the argument be long-only. Also nitpicking the description a bit
```suggestion
    #[clap(long, help = "Disables writing the program's log to a .log file. Don't use if you need help.")]
```

Reply:
done